webapp/schemas.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+from typing import Tuple
+
+from pydantic import BaseModel
+
+
+class Delivery(BaseModel):
+    timestamp: datetime
+    dimensions: Tuple[int, int]
+
+
+
+# class DateTimeMixin(object):
```

Comment:
what is this file for? And why are there so many commented loc?

Reply:
Sorry, added this by mistake. I'd initially created serializers using pydantic - I'll remove it.